packages/csv-stringify/lib/api/index.js

```diff
@@ -125,7 +125,7 @@ const stringifier = function(options, state, info){
         }else{
```

Comment:
Options require to be normalized and/or validated inside `./lib/normalize_options.js`.

Reply:
Done